datafusion/src/physical_plan/parquet.rs

```diff
@@ -27,14 +27,14 @@ use crate::{
     logical_plan::{Column, Expr},
     physical_optimizer::pruning::{PruningPredicate, PruningStatistics},
     physical_plan::{
-        common, DisplayFormatType, ExecutionPlan, Partitioning, SendableRecordBatchStream,
```

Comment:
I really like how the statistics and schema related code has been moved out of `physical_plan` and into` datasource`